dev/io.openliberty.data.internal_fat_nosql/test-applications/DataNoSQLApp/src/test/jakarta/data/nosql/web/DataNoSQLServlet.java

```diff
@@ -75,4 +83,48 @@ public void testBasicQuery() throws Exception {
                                      .sorted(Comparator.naturalOrder())
                                      .collect(Collectors.toList()));
     }
+
+    @Test
+    public void testGeneratedMetaModelsAvailable() {
+        //Check for _Employees is not needed as Jakarta process repositories in a way different than hibernate
+        try {
+            Class.forName("test.jakarta.data.nosql.web._Employee");
+        } catch (ClassNotFoundException e) {
+            fail("Static metamodel class _Employee (for Data) was not generated or available at runtime.");
+        }
+    }
+
+    @Test
+    public void testEmployeeBasicRepositoryFindAllWithPages() {
+        Queue<Integer> wageList = new LinkedList<>();
+        wageList.offer(934500);
+        wageList.offer(904500);
+        wageList.offer(867200);
+        wageList.offer(800500);
+        wageList.offer(734500);
+        wageList.offer(720300);
+        wageList.offer(934572);
+        wageList.offer(934572);
+        wageList.offer(934572);
+
+        PageRequest request = PageRequest.ofSize(3);
+        Order<Employee> order = Order.by(_Employee.wage.desc());
+
+        Page<Employee> page = Employees.findAll(request, order);
```

Comment:
Use the repository instance, not the interface to make method calls.
```suggestion
        Page<Employee> page = employees.findAll(request, order);
```